### PR TITLE
e2e test improvements: check that we found IP address; more debugging…

### PR DESCRIPTION
**Pull request by G-Harmon**
… output.

@nikhiljindal

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/k8s-multicluster-ingress/171)

### CODE REVIEW

File: test/e2e/cases/utils.go [hunk 1]
```diff
@@ -273,8 +275,8 @@ func initDeps() (project, kubeConfigPath, lbName, ipName string, clients map[str
 	glog.Infof("Creating an mci named '%s' with ip address named '%s'", lbName, ipName)
 
 	// Reserve the IP address.
-	if _, err := kubeutils.ExecuteCommand([]string{"gcloud", "compute", "addresses", "create", "--global", ipName}); err != nil {
-		glog.Fatalf("Error creating IP address: %v", err)
+	if output, err := kubeutils.ExecuteCommand([]string{"gcloud", "compute", "addresses", "create", "--global", ipName}); err != nil {
+		glog.Fatalf("Error creating IP address: %v (output: %s)", err, output)
```
Comment (nikhiljindal) [inline test/e2e/cases/utils.go]: err from ExecuteCommand should already have the output?

**Comment by nikhiljindal** [comment]
Ping @G-Harmon.
Wanted to make sure you saw the comment?

**Comment by G-Harmon** [comment]
yes, I saw the comment. I just haven't had time to back and verify the behavior.

File: test/e2e/cases/utils.go (hunk 1 above)
Comment (G-Harmon) [reply test/e2e/cases/utils.go]: Ah, ExecuteCommand does include 'output' in the error. BUT, the actual error from gcloud is not printed. I think we need to look at STDERR. (which is a problem I will leave for later.)

**Comment by G-Harmon** [comment]
PTAL.

**Pull request merged**